gatsby-config.js

```diff
@@ -101,5 +101,14 @@ module.exports = {
       },
     },
     'gatsby-plugin-react-helmet',
+    {
+      resolve: 'gatsby-plugin-sentry',
+      options: {
+        dsn: process.env.SENTRY_DSN,
```

Comment:
@magnificode @racedale @domitriusclark ☝️

Reply:
I made a Sentry account in our 1Password should be able to go here and grab the DSN key
 https://sentry.io/settings/rebuild-black-business-m2/projects/rbb-website/keys/

Reply:
DSN variable added to Netlify!